qiskit/providers/ibmq/ibmqbackend.py

```diff
@@ -78,20 +79,28 @@ def run(self, qobj):
 
         return job
 
-    def properties(self, refresh=False):
+    def properties(self, refresh=False, datetime_filter=None):
         """Return the online backend properties.
 
         Args:
             refresh (bool): if True, the return is via a QX API call.
                 Otherwise, a cached version is returned.
+            datetime_filter (datetime.datetime): datetime to filter properties.
```

Comment:
The docstring should be clearer about what the user should expect. For example, by specifying a filter, this function will return an instance of the backend properties whose timestamp is closest to, but less than the specified datatime filter.